tests/auth_tests/models/custom_user.py

```diff
@@ -100,6 +100,7 @@ def __exit__(self, exc_type, exc_value, traceback):
 class CustomUserWithoutIsActiveField(AbstractBaseUser):
     username = models.CharField(max_length=150, unique=True)
     email = models.EmailField(unique=True)
+    related_users = models.ManyToManyField("self")
```

Comment:
Please reuse the custom used model defined in `tests/auth_tests/models/with_many_to_many.py`.

Reply:
Don't know how I missed that! Done.